### PR TITLE
Add ES6 default parameters.

### PR DESCRIPTION
**Pull request by daveschumaker**

### CODE REVIEW

**Comment by erunion** [comment]
Broken tests!

**Comment by daveschumaker** [comment]
Whatever.

File: src/index.js [hunk 1]
```diff
@@ -276,11 +271,7 @@ class showtimes {
    *                             parsing.
    * @return {object}            Standardized response for the parsed movie.
    */
-  _parseMovie ($, movie, alternate, movieId) {
-    if (typeof alternate === 'undefined') {
-      alternate = false
-    }
-
+  _parseMovie ($, movie, alternate = false, movieId) {
```
Comment (baileylo) [inline src/index.js]: You shouldn't put optional parameters before required parameters.

Comment (daveschumaker) [inline src/index.js]: Ah, good catch!

**Comment by daveschumaker** [comment]
```
/home/travis/build/erunion/showtimes/src/index.js:16
  constructor (location, options = {}) {
                                 ^
SyntaxError: Unexpected token =
```

Going to decline this PR. Looks like current versions of Node don't fully support ES6 yet.

**Pull request closed without merging**